src/guake/terminal.py

```diff
@@ -74,8 +74,8 @@
      r"^[-a-zA-Z0-9\/\_\.\ ]+\.?[a-zA-Z0-9]+\:[0-9]+",
      r"^(.*)\:([0-9]+)"),
     ("line containing '/home/' absolute path",
-     r"(/home/[a-zA-Z0-9_\-\./]+)(:[0-9]+)?",
-     r"(/home/[a-zA-Z0-9_\-\./]+):?([0-9]+)?")
+     r"(/home/[-a-zA-Z0-9_\./]+)(:[0-9]+)?",
```

Comment:
This page http://www.regular-expressions.info/charclass.html :
> The closing bracket ], the caret ^ and the hyphen -  can be included by escaping them with a backslash, or by placing them in a position where they do not take on their special meaning. **The POSIX and GNU flavors are an exception. They treat backslashes in character classes as literal characters. So with these flavors, you can't escape anything in character classes.**

looks like python-vte uses POSIX regex